### ISSUE TITLE
EmrOnEks stack failing for deployments on us-east-2 due to S3 bucket ACL error

### ISSUE THREAD

**Issue by SercanKaraoglu** [aws-samples/aws-analytics-reference-architecture#623]
EmrOnEks stack deployments are failing on us-east-2 region with an error related to the S3 bucket's ACL configuration. The error log indicates that the bucket's ACL cannot be set with ObjectOwnership's BucketOwnerEnforced setting, and the deployment fails with status CREATE_FAILED. This issue is likely caused by the recent security changes introduced by Amazon S3 in April 2023, as mentioned in this blog post: https://aws.amazon.com/blogs/aws/heads-up-amazon-s3-security-changes-are-coming-in-april-of-2023/.

```
7:31:16 PM | CREATE_FAILED        | AWS::S3::Bucket                       | s3accesslogsF93825E5
Bucket cannot have ACLs set with ObjectOwnership's BucketOwnerEnforced setting (Service: Amazon S3; Status Code: 400; Error Code: InvalidBucketAclWithObjectOwnership; Request ID: 8K2NRH
```

To resolve this issue, the EmrOnEks stack's S3 bucket ACL configuration needs to be updated to comply with the new security changes.

**Comment by alexvt-amz** [comment]
Adding `objectOwnership: s3.ObjectOwnership.OBJECT_WRITER ` property to every S3 bucket that gets created by ARA would fix it

**Comment by vgkowski** [comment]
Solved by #624
